### ISSUE TITLE
Keyboard navigation doesn't work in search results

### ISSUE THREAD

**Issue by xiphos-bug-inhaler** [crosswire/xiphos#86]
You can view search results (verse list) only with mouse. If 
you try to operate the verse list with keyboard it's not 
possible to select a verse. 

I propose that arrow up and arrow down keys - that are 
used for navigation in the verse list - may change the bible 
text in the textbox below the list. Enter could make the 
selected verse active in main bible text view. 

Gnomesword is interesting piece of software! 

Reported by: timosa

Original Ticket: "gnomesword/bugs/85":https://sourceforge.net/p/gnomesword/bugs/85
